tools/video_tools/pyscenedetect_shot_detection.xml

```diff
@@ -0,0 +1,20 @@
+<tool id="PySceneDetect_shot_detection" name="PySceneDetect" version="1.0.0">
+  <description>Runs shot detection on a video</description>
+    <requirements>
+    </requirements>
+  <command detect_errors="exit_code"> $__tool_directory__/pyscenedetect_shot_detection.py $input $threshold $amp_shots $frame_stats</command>
+  <inputs>
+	<param name="input" type="data" format="video" label="Video File" help="A video file input for shot detection"/>
+	<param name="threshold" type="integer" label="(Optional) Threshold" value="30" optional="true" help="Sensitivity of the shot detection. The default is 30"/>
+  </inputs>
+  <outputs>
+    <data format="json" name="amp_shots" />
+    <data format="csv" name="frame_stats" />
```

Comment:
If you can use TSV. Galaxy has so many TSV tools already and so many build-in visualisations, this will hopefully make your life easier in the long run.